Content.Server/Atmos/Piping/Unary/Components/GasVentPumpComponent.cs

```diff
@@ -135,6 +135,9 @@ public float InternalPressureBound
         [ViewVariables(VVAccess.ReadWrite)]
         [DataField("depressurizePressure")]
         public float DepressurizePressure = 0;
+
+        [DataField("pressureLockoutOverride")]
```

Comment:
still dont need to specify the tag as its just camelCased variable name, literally `[DataField]` nothing else

Reply:
yeehaw